### ISSUE TITLE
Issue with "Clipboard.write()": example setClipboard reports an error

### ISSUE THREAD

**Issue by MarvinXu** [mdn/content#5566]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/write

#### What information was incorrect, unhelpful, or incomplete?
The first example function setClipboard is incorrect, and reports an error on Chrome.

#### Specific section or headline?
https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/write#example

#### What did you expect to see?
no error. 
This article seems right: [ClipboardItem JavaScript API](https://www.javascripture.com/ClipboardItem)
I made a little change to the example:
```js
function setClipboard(text) {
  const type = 'text/plain';
  let blob = new Blob([text], {type});
  let data = [new ClipboardItem({ [type]: blob })];

  navigator.clipboard.write(data).then(function() {
    /* success */
  }, function(e) {
    /* failure */
  });
}
```

#### Did you test this? If so, how?
I tested on Chrome Version 91.0.4472.77 (Official Build) (64-bit)
![image](https://user-images.githubusercontent.com/7382357/120431175-b72d7b80-c3aa-11eb-8e07-78851b5d7b94.png)

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/clipboard/write`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/write
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/api/clipboard/write/index.html
* Last commit: https://github.com/mdn/content/commit/e085698aeab231f6d0c1d3bb7ea6288237946703
* Document last modified: 2021-05-29T04:16:39.000Z

</details>

**Comment by jpmedley** [comment]
@tomayac brought this to my attention about three weeks back. There's a [webkit ticket](https://bugs.webkit.org/show_bug.cgi?id=222262) with details about why this breaks on Chrome. We have [an open bug](https://crbug.com/1014310) to fix this. Here's the workaround Thomas came up with. I just haven't had time to open a PR.

```js
copyButton.addEventListener('click', async () => {
  try {
    // Safari treats user activation differently:
    // https://bugs.webkit.org/show_bug.cgi?id=222262.
    navigator.clipboard.write([
      new ClipboardItem({
        'text/plain': new Promise(async (resolve) => {
          const svg = svgOutput.innerHTML;
          resolve(new Blob([svg], { type: 'text/plain' }));
        }),
      }),
    ]);
  } catch {
    // Chromium
    const svg = svgOutput.innerHTML;    
    const blob = new Blob([svg], { type: 'text/plain' });
    navigator.clipboard.write([
      new ClipboardItem({
        [blob.type]: blob,
      }),
    ]);
  }
});
```

**Comment by MarvinXu** [comment]
I noticed this [bug](https://crbug.com/1014310) has been resolved, and tested again on Chrome 98.0.4758.81 without any error. May well close this issue.
